lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -1,16 +1,28 @@
+require "pathname"
+
 module CC
   module Analyzer
     class IssueRelativePathValidation < Validation
       def valid?
-        path && !path.start_with?("/")
+        path &&
+          !path.start_with?("/") &&
+          File.exist?(path) &&
```

Comment:
Where did we land on dependent validations? Assuming this validation runs after the existence-validation, and we're OK with relying on order-dependcies, I'd say we do nothing here.